sos/collector/clusters/juju.py

```diff
@@ -156,9 +158,17 @@ def _get_model_info(self, model_name):
 
         return index
 
+    def _get_juju_version(self):
+        """Grab the version of juju"""
+        cmd = "juju version"
+        res = sos_get_command_output(cmd)
```

Comment:
Why not single

    res = sos_get_command_output("juju version")

?

(or maybe even merge it with `return`, though that might sacrifice code readability too much)